.github/workflows/lint.yaml

```diff
@@ -0,0 +1,24 @@
+name: Lint
+
+on:
+  # Allow calling this workflow from other workflows
+  workflow_call:
+  # Allow triggering this workflow manually from the github UI
+  workflow_dispatch:
+  # Lint pull requests
+  pull_request:
+    branches:
+      - master
+      - qa
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Temporarily change swan-cern to use local file://../swan
+        uses: mikefarah/yq@1f0881fb5faf371694bfa108753cda0b824f5037 # v4.28.2
+        with:
+          cmd: yq -i '(.dependencies[] | select(.name == "swan")).repository = "file://../swan"' swan-cern/Chart.yaml
+      - run: helm dependency update swan && helm lint swan && helm package swan
```

Comment:
IIUC here we:
1. Download dependencies and generate lock file
2. Check the sanity of the chart
3. Generate a local packaged chart (not yet pushed to Harbor)

Reply:
yes, `helm package` generates a tar file, but here the output is not really used, instead just tests that it packages correctly.

I borrowed this from the existing Gitlab CI pipeline https://github.com/swan-cern/swan-charts/blob/0c42fda15b943492387bd8cd0c60dcdc17f9628d/.gitlab-ci.yml#L17

Reply:
Ok thanks, here you answer about the package. In the end here you are just checking that the chain update - lint - package runs without problems, but you are not reusing the products of update and package anywhere else.

I would explain this (the ultimate purpose of the workflow) as a comment at the beginning. And perhaps instead of calling it just "lint" it should be called "check_charts" or "charts_sanity_test" or similar?